application/features/bootstrap/MethodContext.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Sil\SilIdBroker\Behat\Context;
+
+use Behat\Gherkin\Node\TableNode;
+use common\models\Method;
+use common\models\User;
+use Webmozart\Assert\Assert;
+
+class MethodContext extends \FeatureContext
+{
+    protected $methodFromDb;
+
+    /**
+     * @Given /^user with employee id (.*) has (?:a|an) (verified|unverified) Method$/
```

Comment:
This is just a suggestion, but these `@Given` (etc.) DocBlock tags have two different syntaxes: regex-based (which PhpStorm generates by default) and a simpler string matching syntax (which Behat generates by default). I find the simpler syntax preferable in most situations.

You're free to use whichever syntax you find works best in any given situation, but I wanted you to at least know about the simpler syntax. For example, the simpler version for this method would be something like this:
```
@Given user with employee id :employeeId has a(n) :verified Method
```

Reply:
Thanks. I think I tried the simpler one but must have done something wrong so I thought I needed the regex version.